demo_nodes_cpp/src/topics/talker.cpp

```diff
@@ -57,7 +57,7 @@ int main(int argc, char * argv[])
 
   while (rclcpp::ok()) {
     msg->data = "Hello World: " + std::to_string(i++);
-    std::cout << "Publishing: '" << msg->data << "'" << std::endl;
+    printf("Publishing: %s\n", msg->data.c_str());
```

Comment:
If we modify this in this PR, it needs to print the exacte same message as before (with single quotes around the data). Otherwise the tests will fail because they expect them:
https://github.com/ros2/demos/blob/0c2fa6b03b96790262932b854646b3fdf21b1bdf/demo_nodes_cpp/test/talker.txt#L1